package/MDAnalysis/units.py

```diff
@@ -236,7 +236,7 @@
 
 
 #: For *time*, the basic unit is ps; in particular CHARMM's
-#: 1 AKMA_ time unit = 4.888821E-14 sec is supported.
+#: 1 AKMA_ time unit = 4.888821e-14 sec is supported.
```

Comment:
To be honest, @nancyalaswad90, I don't think this is really a typo. Both `E` and `e` are valid notations. We would probably want to have a more significant contribution than this.

Reply:
> To be honest, @nancyalaswad90, I don't think this is really a typo. Both `E` and `e` are valid notations. We would probably want to have a more significant contribution than this.



Dear @lilyminium 

I will change the addition to the documentation  and make it more organized 

But About the typo , It is required  as you can see from image below , the `charge` never display in capital letter ad `E`  in any reference.

![image](https://user-images.githubusercontent.com/36210723/163712460-105a1c85-8a82-42b7-9ddf-16d63c6e5240.png)


.

> **So ,please re-check this idea**

Reply:
That `e` is different from the `e` used here. In the image you have shown, the `e` stands for "elementary charge" -- it's an atomic unit for Coulombs of charge. In the documentation that you have changed, the `e` stands for `x 10^`. It is scientific notation. In scientific notation, both `E` and `e` are accepted.

Reply:
**Thanks for clearing things to me @lilyminium** , I will try to find something else to stay eligible for Outreach scholarship